public/app/features/datasources/DataSourceDashboards.test.tsx

```diff
@@ -1,16 +1,17 @@
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 import React from 'react';
 
-import { DataSourceSettings, NavModel } from '@grafana/data';
+import { DataSourceSettings } from '@grafana/data';
 import { getRouteComponentProps } from 'app/core/navigation/__mocks__/routeProps';
+import { getNavModel } from 'app/core/selectors/navModel';
 import { PluginDashboard } from 'app/types';
 
 import { DataSourceDashboards, Props } from './DataSourceDashboards';
 
-const setup = (propOverrides?: object) => {
+const setup = (propOverrides?: Partial<Props>) => {
   const props: Props = {
     ...getRouteComponentProps(),
-    navModel: {} as NavModel,
+    navModel: getNavModel('nav-index', `datasource-dashboards-${'testuid'}`),
```

Comment:
This doesn't work type-wise, you can run `yarn start` locally to see if there are any type errors.

Reply:
Thanks, sorted 🙂